add: add exchange faq tips (#1327)
vesoft-inc/nebula-docs-cn: docs-2.0/nebula-exchange/ex-ug-FAQ.md

-c application.conf
```

### Q：Hive 数据导入时提示 schema 版本不一致

> Spark 日志提示下面这类信息 `Hive Schema version 1.2.0 does not match metastore's schema version 2.1.0 Metastore is not upgraded or corrupt` 
> 
> 这是因为 Hive 环境中配置的 metastore schema 版本和 Spark 使用的 metastore 版本不一致。

Reviewer: ```suggestion
Spark 日志提示 `Hive Schema version 1.2.0 does not match metastore's schema version 2.1.0 Metastore is not upgraded or corrupt` 的原因是 Hive 环境中配置的 metastore schema 版本和 Spark 使用的 metastore 版本不一致。
```

或者：

使用 Nebula Exchange 导入 Hive 数据时可能遇到如下错误提示：
`Hive Schema version 1.2.0 does not match metastore's schema version 2.1.0 Metastore is not upgraded or corrupt`
错误原因为Hive 环境中配置的 metastore schema 版本和 Spark 使用的 metastore 版本不一致。

解决方法：

> 1. 将 Hive 环境中存储 Hive metastore 信息的 MySQL version 信息更新为 Spark 中使用的 metastore 版本：

Reviewer: ```suggestion
1. 将 Hive 环境中存储 Hive metastore 信息的 MySQL version 信息更新为 Spark 中使用的 metastore 版本。
```

冒号不可嵌套。

假设 Hive 在 MySQL 中存储 metastore 的数据库是`hive`，则需要按如下方式修改 hive.VERSION 表中的 version 字段：

```
update hive.VERSION set SCHEMA_VERSION="2.1.0" where VER_ID=1

> ```

Reviewer: 从属于第1条，所以要有2格缩进。
文件名或字段名要用行内代码格式。

> 2. 在 Hive 环境的 hive-site.xml 中增加如下配置：

Reviewer: ```suggestion
2. 在 Hive 环境的 `hive-site.xml` 文件中增加如下配置：
```

```
<property>
   <name>hive.metastore.schema.verification</name>
   <value>false</value>
</property>

> ```

Reviewer: 缩进

> 重启 Hive 即可。

Reviewer: 这也是一个操作步骤

Author: 周老师，我修改完了
